content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade-phase.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Fase de atualização do kubeadm
+weight: 90
+content_type: conceito
+---
+
+Na versão v1.15.0, o kubeadm introduziu um suporte preliminar para as fases `kubeadm upgrade node`. Fases para outros subcomandos `kubeadm upgrade`, tal como `apply`, podem ser adicionadas nas seguintes versões.
+
+## Fase de atualização do nó do kubeadm {#cmd-node-phase}
+
+Usando essa fase, você pode optar por executar as etapas separadas da atualização do plano de controle secundário ou nós. Observe que `kubeadm upgrade apply` ainda precisa ser chamado em um nó principal do plano de controle.
```

Comment:
A couple of issues with this paragraph:
- "upgrade of secondary control-plane or worker nodes" might be better translated with something along the lines of "atualização de nós, sejam eles nós secundários da camada de gerenciamento ou nós de execução de cargas de trabalho". I'm sure there are more options to express this, but I couldn't think of more alternatives right now.
- "camada de gerenciamento" in lieu of other occurrences of "plano de controle".

Reply:
I probably got lost on git, because I remember fixing "camada de gerenciamento" on all files. I will review my PRs. I'm sorry for that.